pkg/api/v1/types.go

```diff
@@ -2011,6 +2013,20 @@ type ComponentStatusList struct {
 	Items []ComponentStatus `json:"items" description:"list of component status objects"`
 }
 
+// DownwardAPIVolumeSource represents a volume containing downward API info
+type DownwardAPIVolumeSource struct {
+	// Items is a list of downward api volume file
+	Items []DownwardAPIVolumeFile `json:"items,omitempty" description:"list of downward api files"`
+}
+
+// DownwardAPIVolumeFile represents information to create the file containing the pod field
+type DownwardAPIVolumeFile struct {
+	// Required: Name is the name of the file
+	Name string `json:"name" description:"the relative path name of the file to be created. Must not be absolute or contain the '..' path. Must be utf-8 encoded."`
```

Comment:
The description is much clearer, thanks.

The field name is not, however. Elsewhere in the API, we use "path" for paths -- look at examples in v1/types.go.